docker/settings.py

```diff
@@ -8,14 +8,36 @@
     DEBUG = False
 
 INSTALLED_APPS += [
-    "oauth_provider",
     "spotseeker_server",
+    "oauth2_provider",
+    "corsheaders",
 ]
 
 MIDDLEWARE += [
     "spotseeker_server.logger.oauth.LogMiddleware",
+    "corsheaders.middleware.CorsMiddleware",
 ]
 
+AUTH_USER_MODEL = "spotseeker_server.Client"
+
+AUTHENTICATION_BACKENDS = (
+    "oauth2_provider.backends.OAuth2Backend",
+    "django.contrib.auth.backends.ModelBackend",
+)
+
+if DEBUG:
+    CORS_ORIGIN_ALLOW_ALL = True
+else:
+    CORS_ORIGIN_ALLOW_ALL = False
+    CORS_ALLOWED_ORIGINS = [
+        "https://scout.uw.edu",
```

Comment:
if os.getenv("ENV", "localdev") == "prod":
    CORS_ALLOWED_ORIGINS = ["https://scout.uw.edu", ...]
else:
    CORS_ALLOWED_ORIGINS = [...]

Reply:
I was thinking that for if ENV is "prod" and if it's "test", but it's probably good to have an ALLOW_ALL = True case for "localdev" so uw-restclients-spotseeker's test-app can access spotseeker-server through anything (e.g. http://icecream.aca.uw.edu:8003/...)

Reply:
Would this work?
```
if DEBUG:  # ENV = localdev
    CORS_ORIGIN_ALLOW_ALL = True
elif os.getenv("ENV", "localdev") == "test":
    CORS_ORIGIN_ALLOW_ALL = False
    CORS_ALLOWED_ORIGINS = [
        "https://test.scout.uw.edu",
        "https://test.manager.scout.uw.edu",
    ]
else:  # ENV = prod
    CORS_ORIGIN_ALLOW_ALL = False
    CORS_ALLOWED_ORIGINS = [
        "https://scout.uw.edu",
        "https://manager.scout.uw.edu",
    ]
```